.github/workflows/InitDev (Postgres), Tests using Sandbox, Year-Specific and Shared Instance.yml

```diff
@@ -11,11 +11,6 @@ on:
   push:
    branches: [main]
   workflow_dispatch:
-    inputs:
```

Comment:
Thanks for doing the cleanup here with the inputs the WriteHost as part of this work